test/cli/autocorrect-kwsplat/test.out

```diff
@@ -1,161 +0,0 @@
-test.rb:17: Cannot call `Object#requires_x` with a `Hash` keyword splat because the method has required keyword parameters https://srb.help/7019
```

Comment:
Ditto the comment below about deleting tests.

Reply:
All of the contents of this test also have been moved to `test/testdata/infer/ruby3_keyword_args.rb`:

https://github.com/sorbet/sorbet/blob/d331f62b15b019c12b751e0c49ac2ae599e93999/test/testdata/infer/ruby3_keyword_args.rb#L40-L68

So we didn't miss any use cases